### PR TITLE
Added -EdgeZone and -ResourceType parameter to New-AzSnapshotUpdateCo…

### PR DESCRIPTION
**Pull request by pselugar**
…nfig and Get-AzComputeResourceSku

## Description
* Added `-EdgeZone` parameter to `New-AzSnapshotUpdateConfig` and `Get-AzComputeResourceSku` cmdlet

Link to Powershell Design PR:  https://github.com/Azure/azure-powershell-cmdlet-review-pr/issues/776
Work Item: https://msazure.visualstudio.com/One/_workitems/edit/9433546

## Checklist

- [x] Check this box to confirm: **I have read the [_Submitting Changes_](../blob/main/CONTRIBUTING.md#submitting-changes) section of [`CONTRIBUTING.md`](../blob/main/CONTRIBUTING.md) and reviewed the following information:**

* **SHOULD** select appropriate branch. Cmdlets from Autorest.PowerShell should go to [`generation`](https://github.com/Azure/azure-powershell/tree/generation) branch. 
* **SHOULD** make the title of PR clear and informative, and in the present imperative tense. 
* **SHOULD** update `ChangeLog.md` file(s) appropriately
    * For any service, the `ChangeLog.md` file can be found at `src/{{SERVICE}}/{{SERVICE}}/ChangeLog.md`
    * A snippet outlining the change(s) made in the PR should be written under the `## Upcoming Release` header in the past tense. Add changelog in description section if PR goes into [`generation`](https://github.com/Azure/azure-powershell/tree/generation) branch.
    * Should **not** change `ChangeLog.md` if no new release is required, such as fixing test case only.
* **SHOULD** have approved design review for the changes in [this repository](https://github.com/Azure/azure-powershell-cmdlet-review-pr) ([_Microsoft internal only_](../blob/main/CONTRIBUTING.md#onboarding)) with following situations
    * Create new module from scratch
    * Create new resource types which are not easy to conform to [Azure PowerShell Design Guidelines](../blob/main/documentation/development-docs/design-guidelines)
    * Create new resource type which name doesn't use module name as prefix
    * Have design question before implementation
* **SHOULD** regenerate markdown help files if there is cmdlet API change. [Instruction](../blob/main/documentation/development-docs/help-generation.md#updating-all-markdown-files-in-a-module)
* **SHOULD** have proper test coverage for changes in pull request.
* **SHOULD NOT** introduce [breaking changes](../blob/main/documentation/breaking-changes/breaking-changes-definition.md) in Az minor release except preview version.
* **SHOULD NOT** adjust version of module manually in pull request

### CODE REVIEW

File: src/Compute/Compute/ChangeLog.md [hunk 1]
```diff
@@ -20,6 +20,9 @@
 
 -->
 ## Upcoming Release
+* Added `-EdgeZone` optional parameter for `Get-AzComputeResourceSku` and `New-AzSnapshotUpdateConfig` cmdlets.
+* Added Disk Delete Optional parameters `OsDisk Deletion Option` and `Delete Option` to the `Set-AzVmssStorageProfile` (OS Disk) and `Add-AzVmssDataDisk` (Data Disk)
```
Comment (wyunchi-ms) [inline src/Compute/Compute/ChangeLog.md:24]: Is this changelog right?

File: src/Compute/Compute/Generated/ResourceSku/ResourceSkuListMethod.cs [hunk 2]
```diff
@@ -44,6 +44,18 @@ public partial class GetAzureRmComputeResourceSku : ComputeAutomationBaseCmdlet
...
+          Mandatory = false,
+          ValueFromPipelineByPropertyName = true,
+          HelpMessage = "Sets the edge zone name. If set, the query will be routed to the specified edgezone instead of the main region.")]
+        public string EdgeZone { get; set; }
+
+        [Parameter(
+          Mandatory = false,
+          ValueFromPipelineByPropertyName = true,
+          HelpMessage = "Sets and Gets Resource Type")]
+        public string ResourceType { get; set; }
```
Comment (wyunchi-ms) [inline src/Compute/Compute/Generated/ResourceSku/ResourceSkuListMethod.cs:57]: `ResourceType` is added but never used. And the document also not updated about it.

File: src/Compute/Compute/ChangeLog.md (hunk 1 above)
Comment (pselugar) [reply src/Compute/Compute/ChangeLog.md:24]: Yes

**Comment by wyunchi-ms** [comment]
Ping @pselugar

**Comment by pselugar** [comment]
> Ping @pselugar

Sorry for the delay, was in talk with the PR design team, removed unused ResourceType .

**Comment by vidai-msft** [comment]
@pselugar Please resolve the conflicts first

**Review by vidai-msft — APPROVED** [review]
Looks good!

**Pull request merged**